packages/dapp/src/apis/civilTCR.ts

```diff
@@ -33,31 +33,31 @@ export async function approveForApply(multisigAddress?: EthAddress): Promise<Two
 
 export async function approveForDeposit(
   amount: number | BigNumber,
-  multisigAddress?: EthAddress,
-): Promise<TwoStepEthTransaction | void> {
+  multisigAddress: EthAddress | undefined,
+): Promise<void | TwoStepEthTransaction> {
   return approve(amount, multisigAddress);
 }
 
-export async function approveForAppeal(): Promise<TwoStepEthTransaction | void> {
+export async function approveForAppeal(): Promise<void | TwoStepEthTransaction> {
   const civil = getCivil();
   const tcr = await civil.tcrSingletonTrusted();
   const government = await tcr.getGovernment();
   const appealFee = await government.getAppealFee();
-  return approve(appealFee);
+  return approve(appealFee, undefined);
 }
 
-export async function approveForChallengeGrantedAppeal(): Promise<TwoStepEthTransaction | void> {
+export async function approveForChallengeGrantedAppeal(): Promise<void | TwoStepEthTransaction> {
   const civil = getCivil();
   const tcr = await civil.tcrSingletonTrusted();
   const government = await tcr.getGovernment();
   const appealFee = await government.getAppealFee();
-  return approve(appealFee);
+  return approve(appealFee, undefined);
 }
 
 export async function approve(
   amount: number | BigNumber,
-  multisigAddress?: EthAddress,
-): Promise<TwoStepEthTransaction | void> {
+  multisigAddress: EthAddress | undefined,
```

Comment:
Shouldn't `multisigAddress?: EthAddress` functionally be the same as `multisigAddress: EthAddress | undefined`? Or is this different in a newer typescript version? Just tested locally and the former is more permissive - you can pass `undefined` or `EthAddress` or nothing at all, but in the latter you must explicitly pass `EthAddress | undefined`.

Seems like the former is more convenient (you can pass `undefined` if you want but you can just leave it off if you don't) and the resulting JS should be the same.

Reply:
I was having trouble getting this to compile and it was just acting crazy (for example (`void | TwoStepEthTransaction` would compile but `TwoStepEthTransaction | void` wouldn't).

I think there might have been some weirdness with the resolve typescript version, which I think i sorted out after I made this change. I just went back and reverted any changes to that file and it appears to be fine. 

So yea, disregard.